From 7d8d9446412b828dc6d8a79a10dc6e9721705315 Mon Sep 17 00:00:00 2001
Message-Id: <7d8d9446412b828dc6d8a79a10dc6e9721705315.1657262743.git.geliang.tang@suse.com>
From: Geliang Tang <geliang.tang@suse.com>
Date: Fri, 8 Jul 2022 14:45:31 +0800
Subject: [PATCH] Revert "Linux 5.10.129"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 7208d1236f72085a3290c41463ee3ee246d8afa5.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7d52cee37488..b89ad8a987db 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 129
+SUBLEVEL = 128
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.34.1

